src/main/java/io/confluent/copycat/hdfs/AvroRecordWriterProvider.java

```diff
@@ -41,7 +41,10 @@
 
     final Schema schema = record.valueSchema();
     final FSDataOutputStream out = path.getFileSystem(conf).create(path);
-    writer.create(avroData.fromCopycatSchema(schema), out);
+    org.apache.avro.Schema avroSchema = avroData.fromCopycatSchema(schema);
+    int version = schema.version();
```

Comment:
Be careful here, version isn't actually required so this could result in a null pointer exception.